thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/controller.js

```diff
@@ -88,14 +112,18 @@ export default Controller.extend({
     function() {
       const severityMap = this.get('severityMap');
       const patternMap = this.get('patternMap');
+      const amountChange = this.get('customPercentChange');
       const selectedPattern = this.get('selectedTunePattern');
       const selectedSeverity = this.get('selectedSeverityOption');
+      const isPercent = selectedSeverity === 'Percentage of Change';
       const mttdVal = Number(this.get('customMttdChange')).toFixed(2);
-      const severityThresholdVal = (Number(this.get('customPercentChange'))/100).toFixed(2);
+      const severityThresholdVal = isPercent ? (Number(amountChange)/100).toFixed(2) : amountChange;
       const featureString = `window_size_in_hour,${severityMap[selectedSeverity]}`;
       const mttdString = `window_size_in_hour=${mttdVal};${severityMap[selectedSeverity]}=${severityThresholdVal}`;
       const patternString = patternMap[selectedPattern] ? `&pattern=${encodeURIComponent(patternMap[selectedPattern])}` : '';
-      return `&features=${encodeURIComponent(featureString)}&mttd=${encodeURIComponent(mttdString)}${patternString}`;
+      const configString = `&features=${encodeURIComponent(featureString)}&mttd=${encodeURIComponent(mttdString)}${patternString}`;
+      console.log('custom string : ', configString);
```

Comment:
Should this console statement still be here?

Reply:
sorry